### PR TITLE
fix: correct websocket docs dep

### PR DESCRIPTION
**Pull request by tallpress**

### CODE REVIEW

File: docs/requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 m2r2
 requests
-websockets
 zeroconf
 dataclasses
 semver
+websocket
```
Comment (walker-j) [inline docs/requirements.txt]: Just to check, we aren't generating this `requirements.txt` from elsewhere, correct?

Comment (LouisBrunner) [reply docs/requirements.txt]: We don't use that library

**Review by LouisBrunner — CHANGES_REQUESTED** [review]

File: docs/requirements.txt (hunk 1 above)
Comment (tallpress) [reply docs/requirements.txt]: we import it in `eva_ws.py`

https://github.com/automata-tech/eva_python_sdk/blob/c6ba57fdfb9116bc5d64b2952104f85d64b39870/evasdk/eva_ws.py#L1

Comment (tallpress) [reply docs/requirements.txt]: used here 
https://github.com/automata-tech/eva_python_sdk/blob/c6ba57fdfb9116bc5d64b2952104f85d64b39870/evasdk/eva_ws.py#L69

Comment (LouisBrunner) [reply docs/requirements.txt]: Check Pipfile for the actual name of the library

**Review by LouisBrunner — APPROVED** [review]

**Pull request merged**